packages/controllers/jest.config.js

```diff
@@ -8,39 +8,21 @@ module.exports = {
   coveragePathIgnorePatterns: ['/node_modules/', '/mocks/', '/test/'],
   coverageThreshold: {
     global: {
-      branches: 67.37,
-      functions: 83.67,
-      lines: 84.6,
-      statements: 84.63,
+      branches: 64.39,
+      functions: 84.07,
+      lines: 84.43,
+      statements: 84.47,
     },
   },
   globals: {
     'ts-jest': {
       tsconfig: 'tsconfig.test.json',
     },
   },
-  projects: [
-    {
-      displayName: 'runner: electron',
-      preset: 'ts-jest',
-      runner: '@jest-runner/electron',
-      // Note that this environment does not support fake timers.
-      testEnvironment: '@jest-runner/electron/environment',
-      testMatch: [
-        '<rootDir>/src/snaps/**/*.test.ts',
-        '<rootDir>/src/services/**/*.test.ts',
-      ],
-    },
-    {
-      displayName: 'runner: default',
-      preset: 'ts-jest',
-      testPathIgnorePatterns: [
-        '<rootDir>/src/snaps/*',
-        '<rootDir>/src/services/*',
-      ],
-      testRegex: ['\\.test\\.(ts|js)$'],
-    },
-  ],
+  preset: 'ts-jest',
+  testRegex: ['\\.test\\.(ts|js)$'],
+  testEnvironment: 'jsdom',
+  testEnvironmentOptions: { resources: 'usable', runScripts: 'dangerously' },
```

Comment:
Is this safe and necessary?

Reply:
These `testEnvironmentOptions` are necessary to run the iframe tests, we may be able to only use these settings for that test though.